### PR TITLE
Ïntroduce multithreading applications to run api and event handler in seperate treads

### PR DESCRIPTION
**Pull request by oysand**

### CODE REVIEW

File: backend/main.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    threads: List[Thread] = []
+
+    populate_mock_db(session=SessionLocal(), connection=connection, base=Base)
+
+    api_thread: Thread = Thread(
+        target=run_app,
+        name="Flotilla Backend API",
+        daemon=True,
+    )
+
```
Comment (anetteu) [inline backend/main.py:19]: remove whiteline

File: backend/main.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    threads.append(api_thread)
+
+    event_handler_thread = Thread(
```
Comment (anetteu) [inline backend/main.py:22]:
```suggestion
    event_handler_thread: Thread = Thread(
```

**Review by aestene — APPROVED** [review]
LGTM

**Comment by aestene** [comment]
Suggestion for documentation purposes would be to change the title of the PR as this is more about introducing multithreaded applications than adding an event handler.

**Pull request merged**